01-introduction.md

```diff
@@ -74,11 +74,11 @@ so that they get the positive feedback of being able to read whole stories as ea
 The basic idea is that once children find reading rewarding,
 they'll be motivated to go back and figure out the letters.
 
-Both approaches are consistent with what we know about how children learn,
-so we need to study them in action to determine which is better.
-The findings have been contradictory:
-while some studies find that phonics works best,
-others show that whole language yields better results.
+These approaches are not equally consistent with what we know about how children
+learn. In fact, the scientific consensus today is that the whole word approach
+was a mistake. Nevertheless, when these methods were studied in classrooms, the
```

Comment:
Links backing up the consensus assertion would be great, especially when followed up by claims of contradictory classroom studies (where links to those studies would also be useful).  Hand-waving about consensus/studies without citing references seems un-sciency ;).